README.md

```diff
@@ -167,6 +167,36 @@ results.each do |test_id, browsers|
   system("rainforest run #{test_id} --browsers #{browsers.join(',')}")
 end
 ```
+
+## Known Issues
+
+### Windows
+
+There is an [open issue](https://github.com/drbrain/net-http-persistent/issues/79) in a dependency of a dependency, which causes the following error on Windows:
+```
+NameError: uninitialized constant Process::RLIMIT_NOFILE
+  from (irb):1
+```
+
+Until a fix is released to that gem, the tempfix is to manually edit that gem's source:
```

Comment:
I would consider monkey patching this in our gem, would that work?

```
Net::Http::Persistent::DEFAULT_POOL_SIZE = ...
```

Reply:
Couldn't get it to work...